pyglet/gui/widgets.py

```diff
@@ -73,6 +73,17 @@ def height(self):
     @property
     def aabb(self):
         return self._x, self._y, self._x + self._width, self._y + self._height
+    
+    # The value property should be overridden in such a way that the widget
+    # can be 'activated' or changed without requiring user input.
+    # The normal events for control activation should NOT be triggered.
+    @property
+    def value(self):
```

Comment:
Rather than having all of the block comments, we should probably just add in the doc strings. Something like: 
"""Query or set the Widget's value.

This properly allows you to set the value of the Widget directly, 
without any user input. This can be useful to restore the Widgets
to a previous state. Note that events are not dispatched when
changing the value directly. 
"""